test.js

```diff
@@ -1,5 +1,5 @@
-import test from 'ava';
```

Comment:
If we could fetch the current test data from the meta repo that would be great (to be able to update tests without updating package), while this is okay too.

Reply:
Sure, I don't see it's a blocking issue since we can update the test later on. So that's why I haven't updated the test yet.